packages/react-components/react-calendar-compat/library/src/components/CalendarDayGrid/CalendarDayGrid.tsx

```diff
@@ -71,6 +71,25 @@ export const CalendarDayGrid: React.FunctionComponent<CalendarDayGridProps> = pr
   const weeks = useWeeks(props, onSelectDate, getSetRefCallback);
   const animateBackwards = useAnimateBackwards(weeks);
   const [getWeekCornerStyles, calculateRoundedStyles] = useWeekCornerStyles(props);
+  const tableRef = React.useRef<HTMLTableElement>(null);
+  const tabster = useTabster();
+
+  // Hack around tabster setting focus on memoized date missing in DOM after month change
```

Comment:
Mind adding more info about what's going on and where is focus lost to in the description?

Also would it be possible to not do this with tabster and instead do it with a callback from that button?

Reply:
Even if it's done as a callback, it's still needs tabster to manage focus

Reply:
Fixed in a different way. Hope this fix is fine